android/src/main/java/com/twiliorn/library/TwilioRemotePreviewManager.java

```diff
@@ -24,6 +24,15 @@ public String getName() {
         return REACT_CLASS;
     }
 
+    @ReactProp(name = "scaleType")
+    public void setScaleType(TwilioVideoPreview view, @Nullable String scaleType){
```

Comment:
Minor but it would be nice if this followed the style in the rest of the file, in particular:

* space between `)` and `{`.
* space between `if` and `(`.
* `if (...)` and `{` on the same line.

Reply:
sure